src/gql/pages/[teamName]/[projectName]/configuration.ts

```diff
@@ -38,6 +39,7 @@ export const CREATE_CONFIGURATION = `mutation CREATE_CONFIGURATION(
   $namePlusTeamName:String!
   $buildCommand:String!
   $openAPISpec:String!
+  $graphQLSchema:String!
```

Comment:
Now that OpenAPI is optional, does it make sense to have `String!` as the input format for both graphql and OpenAPI?